### PR TITLE
CLI support for vcenter

### LINKED ISSUE #247: Update credential CLIs to handle vcenter type
## Specify type:
- Enhancement

## Description:
`qpc cred add --name <cred_name> --type vcenter  --username <username> --password`

`qpc cred edit --name <cred_name>  --username <username> --password`

Handle new type. Setup appropriate exclusion.

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #247

### CODE REVIEW

File: docs/source/man.rst [hunk 1]
```diff
@@ -306,8 +306,8 @@ The following options are available for every Quipucords command.
 Examples
 --------
 
-:Creating a new credential with a keyfile: ``qpc cred add --name=new-creds --username=qpc-user --sshkeyfile=/etc/ssh/ssh_host_rsa_key``
-:Creating a new credential with a password: ``qpc cred add --name=other-creds --username=qpc-user-pass --password``
+:Creating a new credential with a keyfile: ``qpc cred add --name=new-creds **--type=** *network* --username=qpc-user --sshkeyfile=/etc/ssh/ssh_host_rsa_key``
```
Comment (chambridge) [inline docs/source/man.rst]: maybe change to say "Creating a new network type credential ...". And also provide an example of creating a vcenter example?

File: docs/source/man.rst [hunk 2]
```diff
@@ -62,7 +62,7 @@ Creating and Editing Credentials
 
 To create a credential, supply SSH credentials as either a username-password pair or a username-key pair. Quipucords stores each set of credentials in a separate credential entry.
 
-**qpc cred add --name=** *name* **--username=** *username* **(--password | --sshkeyfile=** *key_file* **)** **[--sshpassphrase]** **[--sudo-password]**
+**qpc cred add --name=** *name* **--type=** *(network | vcenter)* **--username=** *username* **(--password | --sshkeyfile=** *key_file* **)** **[--sshpassphrase]** **[--sudo-password]**
```
Comment (chambridge) [inline docs/source/man.rst]: Need to add type flag explanation below.

File: qpc/cred/add.py [hunk 3]
```diff
@@ -62,17 +70,41 @@ def __init__(self, subparsers):
     def _validate_args(self):
         CliCommand._validate_args(self)
 
-        if self.args.filename:
-            # check for file existence on system
-            self.args.filename = validate_sshkeyfile(self.args.filename,
-                                                     self.parser)
+        has_type = 'type' in self.args
```
Comment (chambridge) [inline qpc/cred/add.py]: no thanks. server side only validation.

File: qpc/cred/edit.py [hunk 4]
```diff
@@ -92,12 +82,42 @@ def _validate_args(self):
             print(_(messages.CRED_DOES_NOT_EXIST % self.args.name))
             sys.exit(1)
 
+        if not(self.args.username or self.args.password or
```
Comment (chambridge) [inline qpc/cred/edit.py]: server side validation only needed.

File: docs/source/man.rst (hunk 2 above)
Comment (kholdaway) [reply docs/source/man.rst]: Done

**Pull request merged**